### DISCUSSION TITLE
cargo install cross get killed

### DISCUSSION THREAD

**Discussion by MigAlServ** [cross-rs/cross#1227] [Q&A]
Hi Guys, having issue installing cross to my laptop. When I run command: cargo install -f cross a process stops fetching at 28.74% and Killed label popped up. Tried around 10 times with -f and without, anyways getting the same issue. Did anyone have the same one or know an answer?

**Comment by Emilgardis** [comment]
This is not a cross issue, I'd recommend trying [official channels](https://www.rust-lang.org/community) for support with rust. 

One recommendation I have though, is to try out [sparse protocol](https://blog.rust-lang.org/inside-rust/2023/01/30/cargo-sparse-protocol.html)
